test_alerts.yaml

```diff
@@ -2,6 +2,23 @@ rule_files:
   - prometheus_alert_rules.yaml
 evaluation_interval: 1m
 tests:
+  - interval: 1m
+    input_series:
+      - series: 'ceph_mon_num_elections{job="rook-ceph-mgr"}'
+        values: '0 1 2 3 4 5 6 7 8 9 10 11 12'
+    alert_rule_test:
+      - alertname: CephMonHighNumberOfLeaderChanges
+        eval_time: 11m
+        exp_alerts:
+         - exp_labels:
+             severity: warning
+             job: rook-ceph-mgr
+           exp_annotations:
+             message: 'Storage Cluster has seen many leader changes recently.'
+             description: 'Ceph Monitor "rook-ceph-mgr": instance  has seen 0.016666666666666666 leader changes recently.'
```

Comment:
Is 0.01666... many leader changes? Lets try to put more realistic sample data may be :)

Reply:
This shows leader changes per second, if you multiply it by 60, which is ~1 leader change per minute.